game-core/src/main/java/games/strategy/triplea/delegate/DiceRoll.java

```diff
@@ -945,15 +953,14 @@ private static DiceRoll rollDiceLowLuck(
       final boolean defending,
       final GamePlayer player,
       final IDelegateBridge bridge,
-      final IBattle battle,
+      final Territory location,
```

Comment:
Converting this one parameter is perhaps of a good single PR that could be split off.